gztextractor/extractors/ministry_amendment_and_table_extractor.py

```diff
@@ -1,7 +1,7 @@
 import json
-from extractors.base_extractor import BaseExtractor
-from mergers.ministry_amendment_table import merge_minister_responses
-from prompts.ministry_prompts import INITIAL_PROMPT
+from .base_extractor import BaseExtractor
+from ..mergers.ministry_amendment_table import merge_minister_responses
+from ..prompts.ministry_prompts import INITIAL_PROMPT
```

Comment:
We should be using package name not .. that is not sustainable.. and not readable... can we fix this better.